packages/patternfly-4/react-core/src/components/Chip/Chip.js

```diff
@@ -7,6 +7,17 @@ import { TimesCircleIcon } from '@patternfly/react-icons';
 import styles from '@patternfly/patternfly-next/components/Chip/chip.css';
 import GenerateId from '../../internal/GenerateId/GenerateId';
 class Chip extends React.Component {
+  constructor(props) {
```

Comment:
can we get rid of this constructor and use class properties? Should be able to since this is a class component. Just do   span = createRef(); and state = {isTooltipVisible: false} right on the class since we are using the babel class properties plugin.